packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -439,18 +439,25 @@ export function useUpdateCctpTransactions() {
 
 type useCctpFetchingParams = {
   l1ChainId: ChainId
+  l2ChainId: ChainId
   walletAddress: `0x${string}` | undefined
   pageSize: number
   pageNumber: number
   type: 'deposits' | 'withdrawals' | 'all'
 }
 export function useCctpFetching({
   l1ChainId,
+  l2ChainId,
   walletAddress,
   pageSize = 10,
   pageNumber,
   type
 }: useCctpFetchingParams) {
+  const { isMainnet, isGoerli } = isNetwork(l1ChainId)
+  const { isArbitrumOne, isArbitrumGoerli } = isNetwork(l2ChainId)
+  const isValidChain =
+    (isMainnet || isGoerli) && (isArbitrumOne || isArbitrumGoerli)
```

Comment:
```suggestion
  const { isL1Mainnet, isL1Goerli } = isNetwork(l1ChainId)
  const { isL2ArbitrumOne, isL2ArbitrumGoerli } = isNetwork(l2ChainId)
  const isValidChainPair =
    (isL1Mainnet && isL2ArbitrumOne || (isL1Goerli && isL2ArbitrumGoerli)
```

Reply:
I agree with the suggestion, though it will be:

```suggestion
  const { isMainnet: isL1Mainnet, isGoerli: isL1Goerli } = isNetwork(l1ChainId)
  const { isArbitrumOne: isL2ArbitrumOne, isArbitrumGoerli: isL2ArbitrumGoerli } = isNetwork(l2ChainId)
  const isValidChainPair =
    (isL1Mainnet && isL2ArbitrumOne) || (isL1Goerli && isL2ArbitrumGoerli)
```